schema.json

```diff
@@ -407,7 +407,7 @@
             "by-name",
             "either"
           ],
-          "default": "by-position"
+          "default": "either"
```

Comment:
the description needs to be fixed just above this: (says `by-position`)

![image](https://user-images.githubusercontent.com/364566/80433257-4795b480-88ab-11ea-9fb6-51d8611741eb.png)